src/flowgger/encoder/passthrough_encoder.rs

```diff
@@ -26,7 +26,11 @@ impl Encoder for PassthroughEncoder {
         if let Some(msg) = record.full_msg {
             // First, if specified, prepend a header
             if self.header_time_format.is_some() {
-                res.push_str(&build_prepend_ts(self.header_time_format.as_ref().unwrap()));
+                let ts = match build_prepend_ts(self.header_time_format.as_ref().unwrap()) {
+                    Ok(ts) => ts,
+                    Err(_) => return Err("Cannot prepend str"),
```

Comment:
can you update this to provide an helpful error ? typically that you cannot add the timestamp to the log header or something like that